corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -533,12 +533,15 @@ def _report_filters(self):
 
         def _make_report_filter(conf):
             col_id = self.data_source_properties[conf["property"]]['column_id']
-            return {
+            ret = {
                 "field": col_id,
                 "slug": col_id,
                 "display": conf["display_text"],
                 "type": filter_type_map[conf['format']]
             }
+            if conf['format'] == 'Date':
+                ret.update({'compare_as_string': True})
```

Comment:
do we not specially type the known date columns like `received_on`? or if we do does that filter somehow not pass through this code?